server/src/main/kotlin/org/web3j/openapi/server/console/RunServerCommand.kt

```diff
@@ -74,7 +86,8 @@ class RunServerCommand : Callable<Int> {
             privateKey = credentials.privateKey,
             walletFile = credentials.walletOptions.walletFile,
             walletPassword = credentials.walletOptions.walletPassword,
-            projectName = projectOptions.projectName
+            projectName = projectOptions.projectName,
+            contractAddresses = ContractAddresses(contractAddresses?.mapValues { Address(it.value) })
```

Comment:
This just doesn't make sense. 

The idea of inheriting from a map is to avoid having a wrapper class, if you have `addresses` then you don't need to inherit.

This should look something like this (if that compiles):

```kotlin
ContractAddresses().apply {
    contractAddresses?.let {
        putAll(it.mapValues { Address(it.value) })
    }
}
```

I would even avoid defining the class `ContractAddresses` if possible.

Reply:
So I keep the inheritance and change it to that ? Or remove the inheritance and have the wrapper class ?

Reply:
I think is better to inherit because in this case we only want to use it as a type alias to keep the generic type info.

You could try even to define it as `typealias ContractAddresses = Map<String, Address>` in the Kotlin way to do that.

Maybe that would work so we don't even need a class.

Reply:
The `typealias` isn't accepted by the binder.